packages/story-editor/src/components/canvas/editLayer.js

```diff
@@ -113,8 +141,21 @@ function EditLayerForElement({ element, showOverflow }) {
         showOverflow={showOverflow}
         overflow={showOverflow ? 'visible' : 'hidden'}
       >
-        <EditElement element={element} />
+        <EditElement
+          editWrapper={hasEditModeMoveable && editWrapper}
+          onResize={onResize}
+          element={element}
+          ref={setEditWrapper}
+        />
       </EditPageArea>
+      {hasEditModeMoveable && editWrapper && (
+        <SingleSelectionMoveable
+          selectedElement={element}
+          targetEl={editWrapper}
+          isEditMode
+          ref={setRef}
+        />
+      )}
```

Comment:
I needed the moveable to exist outside of the page area (which is the full-bleed boundary) and have coordinates relative to the entire workspace, so had to move it out here.